okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/DispatcherTraceTerminatingInterceptor.java

```diff
@@ -16,19 +16,18 @@
 
 package com.palantir.conjure.java.okhttp;
 
-import com.palantir.tracing.AsyncTracer;
 import java.io.IOException;
 import okhttp3.Interceptor;
 import okhttp3.Response;
 
 public final class DispatcherTraceTerminatingInterceptor implements Interceptor {
     @Override
     public Response intercept(Chain chain) throws IOException {
-        AsyncTracer tracerTag = chain.request().tag(AsyncTracer.class);
-        if (tracerTag == null) {
+        AsyncTracerTag tracerTag = chain.request().tag(AsyncTracerTag.class);
+        if (tracerTag == null && tracerTag.asyncTracer() == null) {
```

Comment:
npe

Reply:
🤦‍♂️

Reply:
I was surprised errorprone didn't catch this, but then I remembered we don't ahve uber's nullaway enabled!